static/flow/views/landing-page-data-set-view.js

```diff
@@ -121,12 +121,12 @@ var LandingPageDataSetView = function(options) {
 
 
                 } else {
-                    addNoDatasetsToMenu(recordedDataHolder);
+                    addDatasetMessageToMenu(recordedDataHolder, "no available datasets");
```

Comment:
Shouldn't these two messages say `"Error loading programs"`?

Reply:
I refactored the code in both program and dataset loads so what we are seeing here is simply the case where I refactored WET code when we are loading a dataset (whereas I *also* added cases to the load program).  This is just an internal cosmetic change and doesn't affect what the user sees.  I should have made that more clear in the commit comment.

Reply:
Ah alright, I see that the changes to this file are just internal changes. The landing page change does affect what a user sees though, since you added a `"Error loading programs"` when there is an error, right? I would expect to have the same behavior here.

Reply:
D'oh alright I realize I mostly just repeated what you were saying. I guess I just like your new message better than the existing one so I'd swap the old one out, but I guess it's not necessary :)

Reply:
Right!  That one should probably be an "error" not a "no datasets."  Will fix!